tests/publiccloud/run_ltp.pm

```diff
@@ -48,11 +51,18 @@ sub instance_log_args
 sub upload_ltp_logs
 {
     my ($self) = @_;
-    my $log_file = Mojo::File::path('ulogs/ltp_log.json');
+    my $log_file = '';
```

Comment:
it is a bit confusing to init something which suppose to be an object with empty string . Can you just drop this line ?